com.note.slayer.yaml

```diff
@@ -0,0 +1,34 @@
+app-id: com.note.slayer
+base-version: "23.08"
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: noteslayer
+separate-locales: false
+rename-desktop-file: noteslayer.desktop
+rename-icon: noteslayer
```

Comment:
so what does this package do since it doesn't have any way to display anything?

Reply:
ItsFlutter-based application which has its own built-in engine for rendering the GUI

Reply:
that was not the question but you addressed the problem.